src/Profilers/MonitorProfiler/Stacks/StacksEventProvider.cpp

```diff
@@ -46,6 +46,7 @@ HRESULT StacksEventProvider::WriteClassData(ClassID classId, const ClassData& cl
         static_cast<UINT64>(classData.GetModuleId()),
         classData.GetToken(),
         static_cast<UINT32>(classData.GetFlags()),
+        classData.GetStackTraceHidden(),
```

Comment:
note: this relies on an implicit cast from `bool` to `UINT32`. If folks would prefer that this become an explicit `static_cast<UINT32>` let me know.

Reply:
I think that's okay. But I want us to make sure that C++ implicit conversion of bool to UINT32 (`unsigned __int32`) (and that's something the native compiler is doing) is the same as .NET's conversion of bool to UInt32 (which is provided by a library, not compiled).

Reply:
The `bool` to `UINT32` implicit conversion would be considered an integral promotion by C++ and so would be mapped to `true -> 1, false -> 0` which is identical to `Convert.ToUInt32(boolean)` which is equivalent to `return boolean ? 1u : 0u`